### PR TITLE
fix(seccomp): Seccomp default to false and ARM related changes

### PR DESCRIPTION
**Pull request by PrimalPimmy**
**Purpose of PR?**:

I noticed there were some syscalls being used by the init container but were blocked, giving this error 
image

- Added another logic in case the seccomp folder has been made at the kubelet path.
- ARM related syscalls are allowed now
- Seccomp has been defaulted to false
- Controller CI changed kubectl wait to kubectl get script

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [x] Bug fix. Fixes #<issue number>
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

**Review by DelusionalOptimist — DISMISSED** [review]
LGTM.
@PrimalPimmy can you also mention some details in the description about the environment that this happened in (distro, kernel etc)

**Comment by PrimalPimmy** [comment]
> LGTM.
@PrimalPimmy can you also mention some details in the description about the environment that this happened in (distro, kernel etc)

This will happen in any env since it's a seccomp issue.

**Comment by DelusionalOptimist** [comment]
> 
> > LGTM.
> @PrimalPimmy can you also mention some details in the description about the environment that this happened in (distro, kernel etc)
> 
> This will happen in any env since it's a seccomp issue.

Alright... Just mentioned so that we have an understanding of why this was not found earlier : )

**Review by daemon1024 — DISMISSED** [review]

File: pkg/KubeArmorOperator/internal/controller/cluster.go [hunk 1]
```diff
@@ -449,26 +449,28 @@ func (clusterWatcher *ClusterWatcher) UpdateKubearmorSeccomp(cfg *opv1.KubeArmor
...
 		for _, ds := range dsList.Items {
-			if cfg.Spec.SeccompEnabled && ds.Spec.Template.Spec.Containers[0].SecurityContext.SeccompProfile == nil {
-				ds.Spec.Template.Spec.Containers[0].SecurityContext.SeccompProfile = &corev1.SeccompProfile{
-					Type:             corev1.SeccompProfileTypeLocalhost,
-					LocalhostProfile: &common.SeccompProfile,
-				}
-				ds.Spec.Template.Spec.InitContainers[0].SecurityContext.SeccompProfile = &corev1.SeccompProfile{
-					Type:             corev1.SeccompProfileTypeLocalhost,
-					LocalhostProfile: &common.SeccompInitProfile,
+			if ds.Spec.Template.Labels[common.SeccompLabel] == "yes" {
```
Comment (PrimalPimmy) [inline pkg/KubeArmorOperator/internal/controller/cluster.go:452]: New logic here

**Review by daemon1024 — DISMISSED** [review]

**Review by rksharma95 — DISMISSED** [review]

**Review by daemon1024 — DISMISSED** [review]

**Review by rksharma95 — DISMISSED** [review]

**Comment by PrimalPimmy** [comment]
Since we cannot check for current kubelet directory yet, there is no way to check if seccomp profiles can be ignored in case current kubelet directory is different other than `/var/lib/kubelet/seccomp` , hence manual disable through config would be needed

**Comment by daemon1024** [comment]
@PrimalPimmy it's disabled by default right?

**Comment by PrimalPimmy** [comment]
@daemon1024 yes.

**Review by daemon1024 — APPROVED** [review]

**Review by rksharma95 — APPROVED** [review]

**Pull request merged**